OJS.Workers.ExecutionStrategies/DotNetCoreCompileExecuteAndCheckExecutionStrategy.cs

```diff
@@ -78,6 +70,56 @@ public class DotNetCoreCompileExecuteAndCheckExecutionStrategy : ExecutionStrate
             return result;
         }
 
+        protected override IExecutionResult<RawResult> ExecuteNonCompetitive(IExecutionContext<string> executionContext)
```

Comment:
I'm not sure if this is the correct place, but naming can be improved here. I can't really tell what `RawResult` is supposed to be. Maybe something like `SimpleResult` would be an improvement?

 Also `ExecuteNonCompetitive` suggests that there is something that defines execution as competitive/non-competitive and I wonder what could that be?

Reply:
I think `SimpleResult` is not clear as well, "raw" implies that no changes or modifications are applied to the result, it is literally the result from the execution of the code. Maybe `CodeExecutionResult` or something in that direction should be considered..